sycl/include/sycl/ext/oneapi/matrix/matrix-unified.hpp

```diff
@@ -440,7 +441,7 @@ inline __SYCL_ALWAYS_INLINE float round_to_tf32(const float &a) {
   tmp_uint += 0x1000u;
   tmp_uint &= 0xFFFFE000u;
   float ret = 0;
-  std::memcpy(&ret, &tmp_uint, sizeof(float));
+  sycl::detail::memcpy(&ret, &tmp_uint, sizeof(float));
```

Comment:
This seems to be a host-only code, should we keep std::memcpy in here?

Reply:
@YuriPlyakhin  - who are you asking? If you believe it is host-only and want me to change it, I'll be happy to do so. But I'm not familiar with this extension.  Maybe @dkhaldi could inveigh?

Reply:
i agree with @YuriPlyakhin , it is host code. we can keep std::memcpy here